### ISSUE TITLE
hmbc and hmqc of nmr-dataset1 CAS#  57946-61-9  produce crash

### ISSUE THREAD

**Issue by nes-cgn** [cheminfo/nmr-displayer#795]
The processed jcamp files of example CAS 57946-61-9 from nmr-dataset1 produce a crash, even though the other files of the set (even h-f correlations) do not cause any problems. The size is not exceptional, I even tried to shrink them further, but it does not change the result, and it happens on all my computers. The files are packed up here as .zip directories, and I would be glad if I could use them by next week for the class
[crahs-hc-files.zip](https://github.com/cheminfo/nmr-displayer/files/5808038/crahs-hc-files.zip)

**Comment by nes-cgn** [comment]
The console messages from firefox are below
![Konsole1](https://user-images.githubusercontent.com/47237343/104443371-a19fec80-5596-11eb-871d-fe0d4c6fcf56.PNG)
![Konsole2](https://user-images.githubusercontent.com/47237343/104443372-a19fec80-5596-11eb-823a-93cd8311fd40.PNG)

**Comment by hamed-musallam** [comment]
@lpatiny @jobo322 
The problem from fromJCAMP in nmr-parser

**Comment by lpatiny** [comment]
Those jcamps do not contain nucleus information for F1

![image](https://user-images.githubusercontent.com/1484241/104583565-91087880-5661-11eb-9f9e-a548fddc13ab.png)

@nes-cgn  What did you special to obtain this jcamp ? The other files have this information

**Comment by nes-cgn** [comment]
Thank you for investigating, Luc. F1 frequency is mentioned in the file on various other places before and I was not aware that this could cause the problem. Unfortunately, I cannot say that I did anything different with those files. They were recorded on our 600 MHz, but some time ago (with an older version of TopSpin). For processing and export, I worked on TopSpin3, but I did this with all the other data sets, too. And they do originate from various different machines (at least 4 or 5). In eda (Topspin), the F1 frequency setting is correct, and even with edasp I can (virtually see) a proper digital wiring for the files.
